de.fu_berlin.inf.dpp.whiteboard/src/de/fu_berlin/inf/dpp/whiteboard/net/WhiteboardManager.java

```diff
@@ -226,8 +231,29 @@ public ISXEMessageHandler getSXEMessageHandler() {
         return controller;
     }
 
+    /**
+     * Instantiates the bridge with the given browser.
+     * <p>
+     * note: it would be best to call this function after the browser has fully
+     * loaded the document to guarantee proper browser function initialisation
+     * 
+     * @param browser
+     *            the browser created in the view which displays the whiteboard
+     *            html document
+     */
+    public void createBridge(IWhiteboardBrowser browser) {
+        if (bridge != null) {
+            bridge.dispose();
+        }
+        bridge = new BrowserSXEBridge(controller, browser);
+        bridge.init();
+    }
+
     private void dispose() {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this method into the anonymous class declared at line 77. ([squid:S3398](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS3398))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=296&issues=AWbew9F0O08i52vdMn37&open=AWbew9F0O08i52vdMn37)

[ISSUE_KEY]: AWbew9F0O08i52vdMn37